pkgs/by-name/proximity_matcher_webservice/package.nix

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
I suggest to rename the package to `proximity-matcher` which is more human friendly.

Reply:
Renamed the package.